README.md

```diff
@@ -185,39 +186,161 @@ Additional prerequisites are provided in the different sections.
+
+```mermaid
+graph TD;
+    TM-Forum-API-->Context-Broker;
+    Context-Broker-->Persistence;
+```
+
 #### Overview and sub-components
+The TM-Forum-API service is a cluster of individual services providing one specific API each, enabling the participant to only run the necessary subset for its use-case.
+Apart from offering CRUD operations on the managed entities, the service also enables the subscription to notifications based on given queries.
 
-> To understand what the sub-components are, how they all work together and how they interact
+The services are stateless and support horizontal scaling, but require an external cache to avoid having inconsistent caches. Inconsistent caches can result from either changes due to calls to the API, or due to notifications for changes reported by the underlying persistence. If run in a single instance mode, a local cache is acceptable but for larger setups a [Redis](https://redis.io/) installation is recommended.
+
+For reasons of convenience, the TM-Forum-API service can be deploying with an [Envoy API proxy](https://www.envoyproxy.io/) which provides the individual APIs via a single service, routed based on the path. Another convenient feature is a [RapiDoc](https://rapidocweb.com/) container, that can be deployed with the TM-Forum-API service that provides a Openapi based API documentation for the deployed services, with the functionality of querying the API too. 
+
+While the requirement for the persistence is to be compliant to the NGSI-LD API v1.6, the currently recommended Context-Broker is [Scorpio](https://github.com/ScorpioBroker/ScorpioBroker), mainly due to good cloud integration and overall support.
 
 #### Infrastructure requirements
 
-> ex. internal endpoint for the TMForum APIs ; external endpoint for the context broker API ; other endpoints between components ?
+The base memory consumption per deployed pod is listed below but is will increase with the amount of traffic, therefor should only be used as a rough estimate.
 
-#### How to deploy
+| Service          | Memory (Mi) |
+|------------------|-------------|
+| TM-Forum-API Pod | 250         |
+| Scorpio          | 400         |
+| Postgis          | 150         |
+| Redis            | 10          |
+|                  |             |
+|                  |             |
 
-> incl. where to find clean, versioned, infrastructure agnostic HELM charts.
+Apart from the database service, no other service will maintain a own persistence, therefor only for this service a persistent volume claim has to be dimensioned.
 
+#### How to deploy
+The recommended and endorsed way of deployment is via the provided helm charts ( optionally wrapped in ArgoCD Applications).
```

Comment:
Although the actual values.yaml is linked further below, could we add a link to the access node GitHub repo itself here?